protos/local/workflow_commands.proto

```diff
@@ -89,6 +90,9 @@ message ScheduleLocalActivity {
     /// a timer before retrying. It should pass the attempt number from a `DoBackoff` activity
     /// resolution.
     uint32 attempt = 4;
+    /// If this local activity is a retry (as per the attempt field) this needs to be the original
+    /// scheduling time (as provided in `DoBackoff`)
+    google.protobuf.Timestamp original_schedule_time = 14;
```

Comment:
Due to how used (never sent across versions), I support re-indexing as done in another proto in this review.

Reply:
You're right. I was just lazy. :(